ui/src/wide_gauge_stat.tsx

```diff
@@ -76,8 +73,19 @@ interface State {
 
 export const
   View = bond(({ name, state: s, changed }: Model<State>) => {
+    const
+      valueNonDigitCharCount = substitute(s.value, s.data).replace(/[0-9]/g, '').length,
+      auxValueNonDigitCharCount = substitute(s.aux_value, s.data).replace(/[0-9]/g, '').length
```

Comment:
Is this really ***Non*** digit count? Seems like the regex counts the digits or maybe I am wrong?

Reply:
Yes, it really is. `replace` function replaces digits matched by regex with empty string `''` so the length of the resulting string is the count of non-digit characters.

Reply:
Could we use a negative regex (all non-numerical characters) `match` instead of `replace`? Would be less mental gymnastics that way

Reply:
Sure. That's a good idea. What I was trying to achieve was to prevent something like this: `valueNonDigitCharCount = substitute(s.value, s.data).length - substitute(s.value, s.data).replace(/[^0-9]/g, '').length

Reply:
Yes, but match solves this.